P/ProxSDP/Compat.toml

```diff
@@ -3,9 +3,15 @@ MathOptInterface = "0.6"
 TimerOutputs = "0.3-0"
 julia = "0.6"
 
+["0.1"]
+Compat = "1-2"
+TimerOutputs = "0.4-0"
+
 ["0.1-0"]
 Arpack = "0.3-0"
-Compat = "1-2"
 MathOptInterface = "0.8.2-0"
-TimerOutputs = "0.4-0"
 julia = "1"
+
+["0.2-0"]
+Compat = "1"
```

Comment:
You probably want to be compatible with Compat 2 too, right? Just like the previous release?

Reply:
yes, thats what I thought I did with: "Compat = "^1""

Reply:
got is meant up to but excluding